packages/events/src/FederatedEventTarget.ts

```diff
@@ -684,4 +700,34 @@ export const FederatedDisplayObject: IFederatedDisplayObject = {
     }
 };
 
+Object.defineProperties(FederatedDisplayObject, {
+    interactive:  {
+        get()
+        {
+            return this._userSetInteractive;
```

Comment:
Being able to set the global default would be very helpful (e.g., `renderer.events.defaultInteraction = 'none'` or `FederatedDisplayObject.defaultInteraction`)

Reply:
I'm struggling to think how this could be implemented without it causing side effects

We don't have access to the `DisplayObjects` constructor, so setting the default value at a fixed point is hard

For example take this potential implementation

```ts
interactive:  {
    get()
    {
        if (!this._hasUserDefinedInteractive)
        {
            this.interactive = EventSystem.defaultInteraction;
        }

        return this._userSetInteractive;
    },
    set(value: boolean | Interactive)
    {
        this._hasUserDefinedInteractive = true;
        this._userSetInteractive = value;
        this._internalInteractive = convertToInteractive(value);
    },
},
```

With the above `interactive` can be a different value depending on when it is accessed, which is not good
e.g.

```ts
EventSystem.defaultInteraction = false;
const graphics = new Graphics();

// if we uncomment this line, the test will pass
// expect(graphics.interactive).toEqual(false);

EventSystem.defaultInteraction = 'dynamic';

const graphics2 = new Graphics();

expect(graphics2.interactive).toEqual('dynamic');

// this will fail because we never accessed the `interactive` property on `graphics` before switching the default
// however it was originally created with the default value of `false` and should not change
expect(graphics.interactive).toEqual(false);
```

Maybe I'm just being dumb here and there is something obvious i am missing 😄

Reply:
Maybe make the default `_userSetInteractive` undefined then `return this._userSetInteractive ?? EventSystem.defaultInteraction` in the getter.

Reply:
Ok I've got something that works as long as we are ok with `interactive` dynamically changing if the user doesn't explicitly set `interaction` themselves

e.g

```ts
EventSystem.defaultInteraction = false;

const graphics = new Graphics();

// nothing has been explicitly set, so it should be false
expect(graphics.interactive).toEqual(false);
expect(graphics._internalInteractive).toEqual('auto');

EventSystem.defaultInteraction = 'dynamic';

// we have now set the default interaction to dynamic
// we have not explicitly set the state, so this pre existing Graphics should update to be dynamic
expect(graphics.interactive).toEqual('dynamic');
expect(graphics._internalInteractive).toEqual('dynamic');

EventSystem.defaultInteraction = true;
const graphics2 = new Graphics();

// we now explicitly set the default interaction to false
graphics.interactive = false;
expect(graphics.interactive).toEqual(false);
expect(graphics._internalInteractive).toEqual('auto');

// we have NOT set the state on this new Graphics, so it should be true
expect(graphics2.interactive).toEqual(true);
expect(graphics2._internalInteractive).toEqual('static');
```

Reply:
Alternative idea: what about adding a Renderer/Application constructor option instead to set the default? Changing the default on the fly feels very weird and probably not that practical to me.